src/wwwroot/controllers/SucceedRegistrationCtrl.js

```diff
@@ -6,16 +6,49 @@
       .controller('SucceedRegistrationCtrl', SucceedRegistrationCtrl);
   
       SucceedRegistrationCtrl.$inject = [
-      'utils'
+      'auth',
+      '$translate',
+      '$location',
+      'vcRecaptchaService'
     ];
   
-    function SucceedRegistrationCtrl(utils) {
+    function SucceedRegistrationCtrl(auth, $translate, $location, vcRecaptchaService) {
         var vm = this;
-        vm.getCurrentLanguage = getCurrentLanguage;
+        var queryParams = $location.search();
+        var email = queryParams['email'];
+        if (!email) {
+            $location.path('/');
+        }
+        vm.resendEmail = resendEmail;
+        vm.resendFinished = false;
+        vm.setResponse = setResponse;
+        vm.setWidgetId = setWidgetId;
+        vm.captchaExpiration = captchaExpiration;
+
+        function setWidgetId (widgetId) {
+            vm.widgetId = widgetId;
+        };
+        
+        function captchaExpiration () {
+            vcRecaptchaService.reload(vm.widgetId);
+            vm.response = null;
+        };
+
+        function resendEmail() {
+            vcRecaptchaService.reload(vm.widgetId);
```

Comment:
It looks weird, Is it necessary just before use captcha? 

If it is necessary, maybe it is a better idea to _reload_ during page initialization in place of doing after the click.